backend/pkg/api/connect/integration/api_suite_test.go

```diff
@@ -53,29 +63,59 @@ func (s *APISuite) SetupSuite() {
 	t := s.T()
 	require := require.New(t)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
 	defer cancel()
 
-	// 1. Start Redpanda Docker container
-	container, err := redpanda.RunContainer(ctx, testcontainers.WithImage("redpandadata/redpanda:v23.2.18"))
-	container.FollowOutput(testutil.TestContainersLogger{LogPrefix: "CONTAINER (connect-integration): "})
+	// 1. setup docker network
+	networkName := "redpanda-net"
+	network, err := setupNetwork(networkName)
+
+	require.NoError(err)
+
+	s.network = network
+
+	// 2. Start Redpanda Docker container
+	container, err := redpanda.RunContainer(ctx, testcontainers.WithImage("redpandadata/redpanda:v23.2.18"), testcontainers.WithNetwork(networkName, "redpanda"))
+	container.FollowOutput(testutil.TestContainersLogger{LogPrefix: "CONTAINER REDPANDA (connect-integration): "})
 	require.NoError(container.StartLogProducer(context.Background()))
 
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Maybe `require.NoError(err)` right after the container started? If redpanda.RunContainer just yields an error you would cause a panic in line 79 where you try to call FollowOutput().

Also this is missing a call to `StartLogProducer()` , in order to print container logs, but I think we usually don't want to print container logs because they are pretty verbose and should usually not be needed